app/models/site_updates/new_concept_update.rb

```diff
@@ -0,0 +1,44 @@
+class SiteUpdates::NewConceptUpdate < SiteUpdate
+  params :concept
+
+  def guard_params
+    "Concept##{concept.id}"
+  end
+
+  def i18n_params
+    {
+      concept_name: concept.name,
+      concept_url: Exercism::Routes.track_concept_url(track, concept),
+      maker_handles: maker_handles
+    }
+  end
+
+  def cacheable_rendering_data
+    super.merge(
+      maker_avatar_urls: makers.map(&:avatar_url)
+    )
+  end
+
+  def icon_type
+    :concept
+  end
+
+  # TODO: This is pretty gross
+  def icon_url
+    concept.name[0, 2]
```

Comment:
Eh, just the first three characters? How does that work?

Should we do something like with exercises:

```ruby
def icon_name
  config[:icon] || exercise_slug.to_s
end
```

(that's defined it `Git::Exercise` btw)

Reply:
Concepts don't have actual icons, the first two chars just get rendered in HTML and decorated in a box.

So this gives us the data for this:

<img width="144" alt="Screenshot 2021-06-15 at 12 44 28" src="https://user-images.githubusercontent.com/286476/122046928-6de02180-cdd7-11eb-82cd-bbd1333bc35e.png">

I feel like my comment here is very accurate 😉

Reply:
I tis :)